src/algorithms/pathfinders/path.ts

```diff
@@ -0,0 +1,33 @@
+import EdgeCost from '../models/edgecost';
+import Vertex from '../models/vertex';
+/**
+ * Path class, stores pathData and costs in a path
+ */
+export default class Path {
+    pathData: Array<{vertex: Vertex, edgecost: EdgeCost}>;
+    constructor() {
+        this.pathData = new Array<{vertex: Vertex, edgecost: EdgeCost}>();
```

Comment:
```suggestion
        this.pathData = [];
```
Same as comment before.